src/isomorphic/classic/class/__tests__/ReactClassMixin-test.js

```diff
@@ -368,24 +372,24 @@ describe('ReactClass-mixin', function() {
   });
 
   it('should include the mixin keys in even if their values are falsy',
-    function() {
-      var mixin = {
-        keyWithNullValue: null,
-        randomCounter: 0
-      };
+      function() {
```

Comment:
What if we had a flag you could pass to `grunt lint` to hide warnings? It wouldn't solve the problem of discouraging people from using long lines of code, but it would at least make it easier to sort through the errors.